<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositcalc</class>
 <widget class="QDialog" name="depositcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(189, 189, 189);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cумма вклада:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_amount">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cрок вклада:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_term">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Процентная ставка:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_interestRate">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hалоговая ставка:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_taxRate">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget_periodOfPayments">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 221, 221);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>319</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Переодичность выплат:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 134, 182);</string>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>321</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_listPays">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(208, 208, 208);</string>
      </property>
      <property name="text">
       <string>Список пополнений:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_addPays">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Добавить пополнение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);</string>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Сумма пополнения</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_removePays">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Удалить запись пополнения</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>321</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="checkBox_Withdrawals">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(208, 208, 208);</string>
      </property>
      <property name="text">
       <string>Cписок частичных снятий:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_addWithdrawals">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Добавить снятие</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_Withdrawals">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);</string>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Сумма снятия</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_removeWithdrawals">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Удалить запись снятия</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Начисленные проценты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_totalInterest">
          <property name="styleSheet">
           <string notr="true">QLabel {
  background-color: rgb(255, 255, 255);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сумма налога</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_totalTax">
        <property name="styleSheet">
         <string notr="true">QLabel {
  background-color: rgb(255, 255, 255);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>сумма депозита на конец срока</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_totalDeposit">
        <property name="styleSheet">
         <string notr="true">QLabel {
  background-color: rgb(255, 255, 255);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
